core/math/camera_matrix.h

```diff
@@ -44,7 +44,7 @@ struct CameraMatrix {
 		PLANE_BOTTOM
 	};
 
-	real_t matrix[4][4];
+	real_t matrix[4][4]{};
```

Comment:
Value inits the 16 elements. Should be fine?

Reply:
init to identity maybe?